src/sinks/honeycomb.rs

```diff
@@ -26,7 +26,7 @@ pub struct HoneycombConfig {
     #[serde(skip, default = "default_endpoint")]
     endpoint: String,
 
-    /// The team key that will be used to authenticate against Honeycomb.
+    /// The team key that is used to authenticate against Honeycomb.
```

Comment:
Unrelated to your change but "team" key seems weird to me. IDK much about Honeycomb but this seems like a typo to me "team" -> "API"

Reply:
> X-Honeycomb-Team, which is your API key.

I think API is a good change, and this is why it got called team.